boa/network.py

```diff
@@ -402,7 +409,11 @@ def deploy(
         print(f"contract deployed at {create_address}")
 
         if (deployments_db := get_deployments_db()) is not None:
-            contract_name = getattr(contract, "contract_name", None)
+            contract_name = (
```

Comment:
why would we want 2 names for the contract? This value `contract_name` should be equal to `contract.contract_name`

Reply:
What do you mean? This would only set 1 name. It defaults to the `filename` unless one is passed.

Reply:
yea i don't think contract_name should be passed to env.deploy(). it already exists on the contract object